kitsune/sumo/static/sumo/js/form-wizard.js

```diff
@@ -225,6 +225,11 @@ export class FormWizard extends HTMLElement {
       let progress =
         Math.ceil((activeStepIndex / (this.#steps.length - 1)) * 100) || 10;
       this.#progressIndicator.value = progress;
+      this.#progressIndicator.setAttribute(
+        "aria-label",
+        gettext(`Step ${activeStepIndex + 1} of 3`)
```

Comment:
I was again surprised by the limitations of even the GNU `xgettext` extractor when working with JavaScript files. I'm sorry to say that it can't yet handle this case. What you'll have to do instead, which is what we do in other places in our JavaScript code, for example [here](https://github.com/mozilla/kitsune/blob/d9b73f19471180f3fbb606c585321d9e7a1c4d75/kitsune/sumo/static/sumo/js/wiki.js#L604), is use a function called `interpolate` that we define [here](https://github.com/mozilla/kitsune/blob/d9b73f19471180f3fbb606c585321d9e7a1c4d75/kitsune/sumo/static/sumo/js/i18n.js#L21):

```suggestion
        interpolate(gettext("Step %s of 3"), [activeStepIndex + 1])
```

I've tested that locally, and it both works as expected, and is extracted properly into our [JavaScript POT file](https://github.com/mozilla-l10n/sumo-l10n/blob/master/templates/LC_MESSAGES/djangojs.pot) that we share with Pontoon.